uniffi/src/templates/macros.rs

```diff
@@ -0,0 +1,20 @@
+{% macro return_type_func(func) %}{% match func.ffi_func().return_type() %}{% when Some with (return_type) %}{{ return_type|ret_type_c }}{%- else -%}(){%- endmatch -%}{%- endmacro -%}
```

Comment:
Defined some common macros for args/return types/errors/ on the Rust side. I'll keep an eye on the effort on #21 as I tackle the backends 😄